### ISSUE TITLE
As an advanced user I'd like taxonomies to be integrated into search

### ISSUE THREAD

**Issue by longhotsummer** [laws-africa/peachjam#749]
to help me dive into those, not just matching docs

**Comment by longhotsummer** [comment]
I see two aspects here:

* showing the taxonomy as a facet
* showing taxonomy entry search results

**Comment by niiroobiro** [comment]
@longhotsummer I want to close this because I believe it is done

**Comment by longhotsummer** [comment]
No, it's not implemented yet. Users can't facet or search taxonomy entries via the search page.
